librz/analysis/block.c

```diff
@@ -1041,29 +1048,35 @@ RZ_API bool rz_analysis_block_set_op_offset(RzAnalysisBlock *block, size_t i, ut
 }
 
 /**
- * @return the address of the instruction that occupies a given offset or UT64_MAX if off is not in the block.
+ * \return the index of the instruction that occupies a given offset or -1 if addr is not in the block.
```

Comment:
Maybe you could rename the variable to `offset` and then use `\p offset` in the Doxygen description?

Reply:
I will change the docs `\p addr` since it is absolute